amora/dash/pages/feature_store.py

```diff
@@ -85,12 +85,18 @@ def layout() -> Component:
         fv.name: fv for fv in store.registry.list_feature_views(store.project)
     }
 
-    feature_views = html.Div(
-        [
-            card_item(model=model, fv=registry_fvs.get(fv.name))
-            for (fv, fs, model) in list(FEATURE_REGISTRY.values())
-        ]
-    )
+    if registry_fvs:
+        feature_views = html.Div(
+            [
+                card_item(model=model, fv=registry_fvs.get(fv.name))
+                for (fv, fs, model) in list(FEATURE_REGISTRY.values())
+            ]
+        )
+    else:
+        feature_views = html.Div(
+            [html.H1("Error loading feature view...")]
+        )
```

Comment:
Temos que escrever um caso de teste pra esse cenário. Algo assim:

```suggestion
    else:
        feature_views = html.Div(
            [html.H1("Error loading feature view...")], id='feature-views-loading-error'
        )
```
Com esse ID, conseguimos validar que quando a página é carregada e `registry_fvs == {}`, a mensagem de erro é exibida:

```python
def test_feature_store_page_error(amora_dash: DashComposite):
    amora_dash.visit_and_snapshot(
        resource_path="/feature-store",
        hook_id="feature-view-loading-error",
        wait_for_callbacks=True,
        stay_on_page=True,
    )
```

Reply:
Adicionei e marquei como skip, assim que tiver o merge do: https://github.com/mundipagg/amora-data-build-tool/pull/176, a gente remove

Reply:
Já podemos remover esse skip e fazer o merge do PR ?